lib/temperature/index.js

```diff
@@ -93,6 +98,7 @@ TemperatureSensor.prototype.start = function start() {
   }
     
   try {
+    if (this.options.controller !== 'simulator') {
```

Comment:
or test calibrate, and other option is add a calibration function in simulator that will run start the interval loop like driver...

Reply:
ok you could test precense of function calibrate
but it's fine that way too